rules/otp.bzl

```
@@ -348,3 +350,108 @@ STD_OTP_OVERLAYS = STD_OTP_OVERLAYS_WITHOUT_SECRET_PARTITIONS + [
     "//hw/ip/otp_ctrl/data:otp_json_secret1",
     "//hw/ip/otp_ctrl/data:otp_json_secret2_unlocked",
 ]
+
+def otp_hex(v):
+    return hex(v)
+
+def otp_bytestring(byte_list):
+    bytestring = "".join(reversed([hex_digits(byte, width = 8) for byte in byte_list]))
+    return "0x{}".format(bytestring)
```

nit: this switches to the `str` domain to early:
```suggestion
    val = 0
    for b in reversed(byte_list):
        val = (val << 8) | b
    return hex(val) # TODO: pass proper width
```

Also, should this expect a width parameter as a consistency check?